src/components/picker/index.js

```diff
@@ -385,10 +392,18 @@ class Picker extends PureComponent {
 
     const textInputProps = TextField.extractOwnProps(this.props);
     const label = this.getLabelValueText();
-    
+    const {paddings, margins, alignments, positionStyle} = modifiers;
+
     return (
       <TextField
         {...textInputProps}
+        containerStyle={[
+          containerStyle,
+          paddings,
+          margins,
+          alignments,
+          positionStyle
```

Comment:
`containerStyle` should be prioritized over modifiers style. 
`alignments` style is redundant in this case, it does't affect the component so there's no need to support it.

Reply:
Done.